src/applications/mhv/medical-records/util/helpers.js

```diff
@@ -194,3 +194,30 @@ export const getNameDateAndTime = user => {
     .format('M-D-YYYY_hhmmssa')
     .replace(/\./g, '')}`;
 };
+
+/**
+ * Helper function to dispatch details id if a matching is found in the list.
+ * If no matching is found, it fetches the details id using getDetail().
+ *
+ * @param {string} id - The ID of the detail to retrieve.
+ * @param {Array} list - The list of detail object to search in.
+ * @param {Function} dispatch - The Redux dispatch function.
+ * @param {Function} getDetail - The function to fetch detail's object by ID from an API.
```

Comment:
could you add docs for actionsGetFromList and actionsGet as well? thanks!

Reply:
Completed